fieldservice_route/models/fsm_order.py

```diff
@@ -14,13 +14,23 @@ class FSMOrder(models.Model):
     def _get_default_person(self):
         return self.fsm_route_id.fsm_person_id.id or False
```

Comment:
This is not going to work, as `fsm_route_id` won't never have a value on default. This must work with context values or previous defaults.